### PR TITLE
[doc][ybm] Changes for pricing plans

### PR DESCRIPTION
**Pull request by ddhodge**
Changes for pricing plans
DOC-206
DOC-190

@netlify /preview/yugabyte-cloud/

### CODE REVIEW

File: docs/content/preview/yugabyte-cloud/cloud-admin/_index.md [hunk 1]
```diff
@@ -13,7 +13,11 @@ menu:
 type: indexpage
 ---
 
-Before you can add Dedicated clusters, you need to create a billing profile and add a payment method. You review your usage, manage your billing profile and payment methods, and review invoices from the **Usage & Billing** page.
+There are no set-up charges or commitments to begin using YugabyteDB Managed. At the end of the month, you are automatically charged for that month's usage.
+
+Review your usage and plan details, manage your billing information, and review invoices from the **Usage & Billing** page.
+
+For information on available plans and their features, refer to [YugabyteDB Managed Standard Price List](https://www.yugabyte.com/yugabyte-cloud-standard-price-list/).
```
Comment (rangasayeelc-yb) [inline docs/content/preview/yugabyte-cloud/cloud-admin/_index.md:20]: Note that the standard price list will redirect to new pricing page. update link to www.yugabyte.com/pricing

File: docs/content/preview/yugabyte-cloud/cloud-basics/create-clusters-overview.md [hunk 2]
```diff
@@ -229,9 +231,7 @@ For more information on users and roles in YugabyteDB Managed, refer to [Databas
 
 The biggest factor in the price of a cluster is the number vCPUs.
 
-Cluster charges are based on the total number of vCPUs used and how long they have been running. Cluster per-hour charges include [free allowances](../../cloud-admin/cloud-billing-costs/) for disk storage, backup storage, and data transfer. If you use more than the free allowance, you incur overages on top of the base vCPU capacity cost.
-
-Before creating a cluster, you need to create your billing profile and add a payment method. Refer to [Manage your billing profile and payment method](../../cloud-admin/cloud-billing-profile/).
+Cluster charges are based on the total number of vCPUs used and how long they have been running. The rate is determined by your [pricing plan](https://www.yugabyte.com/pricing/).
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-cloud/cloud-basics/create-clusters-overview.md:234]:
```suggestion
Cluster charges are based on the total number of vCPUs used and how long they have been running. The rate is determined by your [pricing plan](https://www.yugabyte.com/yugabytedb-managed-standard-price-list/).
```

File: docs/content/preview/yugabyte-cloud/managed-automation/managed-cli/managed-cli-examples/managed-cli-example-create.md [hunk 3]
```diff
@@ -22,7 +22,7 @@ This guide assumes you have already done the following:
 - [Installed ybm CLI](../../../managed-cli/managed-cli-overview/#install-ybm).
 - [Configured ybm CLI](../../../managed-cli/managed-cli-overview/#configure-ybm) with your API key.
 
-Note that you can only create one Sandbox cluster per account. To create VPCs and dedicated clusters, you also need to add a [billing profile](../../../../cloud-admin/cloud-billing-profile/) and payment method, or you can [request a free trial](../../../../managed-freetrial/). For more information, refer to [What are the differences between Sandbox and Dedicated clusters](../../../../../faq/yugabytedb-managed-faq/#what-are-the-differences-between-sandbox-and-dedicated-clusters).
+Note that you can only create one Sandbox cluster per account. To create VPCs and dedicated clusters, you need to [choose a plan](https://www.yugabyte.com/pricing/), or you can [start a free trial](../../../../managed-freetrial/). For more information, refer to [What are the differences between Sandbox and Dedicated clusters](../../../../../faq/yugabytedb-managed-faq/#what-are-the-differences-between-sandbox-and-dedicated-clusters).
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-cloud/managed-automation/managed-cli/managed-cli-examples/managed-cli-example-create.md:25]:
```suggestion
Note that you can only create one Sandbox cluster per account. To create VPCs and dedicated clusters, you need to [choose a plan](https://www.yugabyte.com/yugabytedb-managed-standard-price-list/), or you can [start a free trial](../../../../managed-freetrial/). For more information, refer to [What are the differences between Sandbox and Dedicated clusters](../../../../../faq/yugabytedb-managed-faq/#what-are-the-differences-between-sandbox-and-dedicated-clusters).
```

File: docs/content/preview/yugabyte-cloud/managed-automation/managed-cli/managed-cli-examples/managed-guide-api.md [hunk 4]
```diff
@@ -37,7 +37,7 @@ YBM_ACCOUNT_ID="<account_ID>"
 YBM_PROJECT_ID="<project_ID>"
 ```
 
-To create VPCs and dedicated clusters, you also need to add a [billing profile](../../../cloud-admin/cloud-billing-profile/) and payment method, or you can [request a free trial](../../../managed-freetrial/).
+To create VPCs and dedicated clusters, you need to [choose a plan](https://www.yugabyte.com/pricing/), or you can [start a free trial](../../../managed-freetrial/).
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-cloud/managed-automation/managed-cli/managed-cli-examples/managed-guide-api.md:40]:
```suggestion
To create VPCs and dedicated clusters, you need to [choose a plan](https://www.yugabyte.com/yugabytedb-managed-standard-price-list/), or you can [start a free trial](../../../managed-freetrial/).
```

**Review by aishwarya24 — COMMENTED** [review]
left some comments.

**Pull request closed without merging**